config/locales/en/publisher_information/request_feature.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  publisher_information:
+    request_feature:
+      title: Request Feature
+      send_us_feedback: You can help us prioritise features by [sending us feedback](https://support.integration.publishing.service.gov.uk/general_request/new).
```

Comment:
Shall we suffix this key with `govspeak` to indicate that it's govspeak. Note this would be a new convention but I think it's a helpful one.

In the same way we use a suffix of _html for html: https://github.com/alphagov/content-publisher/blob/master/config/locales/en/documents/index.yml#L16

Another idea would be that since this is simple HTML would be to just have these as html and skip the govspeak.

Reply:
I've removed the Request Feature partial view and the YML file that goes with it in favour of just putting the content into a request_feature.govspeak.md file.  I've done that for two reasons (1) I think that chunk is more manageable by the content developers because it lets them edit it in the same way they do other main parts of the view text and (2) it helps cut down on how many YAML messages we have containing mark-up text.  We probably can't avoid this but I'd like to minimise it